packages/frontity/src/steps/index.ts

```diff
@@ -310,6 +310,25 @@ export const downloadFavicon = async (path: string) => {
   await new Promise((resolve) => fileStream.on("finish", resolve));
 };
 
+/**
+ * Creates a .gitignore file and initializes git.
+ *
+ * @param path - The path where git should be initialized.
+ */
+export const initializeGit = async (path: string) => {
+  const fileTemplate = await readFile(
+    resolvePath(__dirname, "../../templates/gitignore-template"),
+    {
+      encoding: "utf8",
+    }
+  );
+  const filePath = resolvePath(path, ".gitignore");
+  await writeFile(filePath, fileTemplate);
+  await promisify(exec)("git init", { cwd: path });
+  await promisify(exec)("git add .", { cwd: path });
+  await promisify(exec)('git commit -m "initial commit"', { cwd: path });
```

Comment:
Should we include some `Frontity` related wording in here? Like:
```
Initialised with Frontity
```
😃

Reply:
I thought about this while writing it. I think it makes sense, so I'll change the message for the one you proposed.